code/UserForm.php

```diff
@@ -325,6 +325,8 @@ public function getSubmissionsGridField()
         // attach every column to the print view form
         $columns['Created'] = 'Created';
         $columns['SubmittedBy.Email'] = 'Submitter';
+        $this->extend('updateSubmissionsGridFieldExportColumns', $columns);
```

Comment:
Would it also work for you if this was `$this->extend('updateSubmissionsGridField', $submissions);` and moved just before the whole gridfield is returned? It should be possible to adjust the columns that way (and reassign them to the gridfield again there) while giving a bit wider adjustability to the whole gridfield, not just the export columns.

Reply:
`updateSubmissionsGridField` would be my preference.